Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/localizer/Localizer.cpp

```diff
@@ -123,9 +123,19 @@ void Localizer::init_R()
     ;
 
     std::stringstream s;
+    XmlRpc::XmlRpcValue v;
```

Comment:
Is there a way for this to just be an array of doubles? Does it have to be this specific class?

Reply:
I'm not totally sure, will have to look into it more - that's just what I tried after looking up a ROS forum on reading values from a launch file

Reply:
Check out the ros docs: http://wiki.ros.org/roscpp/Overview/Parameter%20Server#Retrieving_Lists
We have kinetic (newer than groovy), so we don't need to use this method

Reply:
resolved - check again?